examples/cms-notion/lib/api.js

```diff
@@ -0,0 +1,24 @@
+const dev = process.env.NODE_ENV !== 'production';
+
+export const server = dev ? 'http://localhost:3000' : 'https://your_deployment.server.com';
+
+export async function getAllPostsForHome() {
```

Comment:
These don't need to be behind API routes, as we can use the logic directly since it's on the server 👍

Reply:
Do you mean moving the logic from here to the api server, in order to minimise the different locations of the response data?